recipe-server/normandy/recipes/api/views.py

```diff
@@ -62,7 +62,7 @@ def retrieve(self, request, name, impl_hash):
 
 
 class RecipeFilters(django_filters.FilterSet):
-    enabled = CaseInsensitiveBooleanFilter(name='enabled', lookup_type='eq')
+    enabled = CaseInsensitiveBooleanFilter(name='enabled', lookup_expr='eq')
```

Comment:
The version of the router that I based my changes on came from the new version of DRF. That version of DRF required django-filters 1.0.